src/FakeClient/FakeClientMain.cpp

```diff
@@ -377,6 +381,7 @@ int main(int argc, char* argv[]) {
     ErrorMessageOr<std::vector<orbit_grpc_protos::ModuleInfo>> modules_or_error =
         orbit_module_utils::ReadModules(options.process_id);
     ORBIT_FAIL_IF(modules_or_error.has_error(), "%s", modules_or_error.error().message());
+    process.UpdateModuleInfos(modules_or_error.value());
```

Comment:
I'm a bit scared of making a change to `FakeClient` without any manual testing. Would you be able to do that?

Reply:
Tested together offline.